src/schema/user.js

```diff
@@ -422,13 +404,36 @@ export const resolvers = {
         )
       );
 
-      // Remove agencyes which doesnt exist in VIP
+      // Remove agencies that dont exist in VIP
       // Example "190976" and "191977" (no VIP info) on testuser Michelle Hoffmann will return empty results
       const filteredAgencyInfoes = agencyInfos.filter(
         (agency) => agency?.result.length > 0
       );
 
-      return filteredAgencyInfoes;
+      const sortedAgencies = filteredAgencyInfoes.sort((a, b) =>
+        a.result[0].agencyName.localeCompare(b.result[0].agencyName)
+      );
+
+      const loginAgencyIdx = sortedAgencies.findIndex((agency) => {
+        const matchingBranch =
+          agency.result.findIndex(
+            (library) => library.branchId === context.smaug.user.agency
+          ) > -1;
+        //problem I: i get loginbibliotek instead of loginagency
+        //problem II: we dont return all bibs for an agency, but only first 20 or so --> can be that loginbib is not included
+        return (
```

Comment:
ift problem II @mtni 
hvordan kan jeg udvide agencies søgning således at den returnerer alle branches for en agency?
alternativt kunne jeg finde agencyId frem fra branchid, som kommer med smaug.user.agency --> har vi en endpoint der finder agency for branchId?

Reply:
Den her må vi lige snakke om sammen senere.. Bare så jeg lige forstår det rigtigt :upside_down_face: